app/localization/km.json

```diff
@@ -70,8 +70,8 @@
   "allParticipant": "អ្នក​ចូល​រួម​សរុប​",
   "enterNumberOfParticipant": "សូមបញ្ចូលចំនួន​អ្នក​ចូលរួម​",
   "allParticipantRequireMsg": "ត្រូវ​ការ​ចំនួន​អ្នកចូល​រួម​",
-  "female": "ស្រី",
-  "enterNumberOfFemale": "សូម​បញ្ចូល​ចំនួន​អ្នក​ចូល​រួមដែលជា​​មនុស្សស្រី​",
+  "female": "នារី",
```

Comment:
@limkimsan please check the other section, we used female(ស្រី), make it consistent

Reply:
Bong, I have finished changing the word female(ស្រី).

Reply:
@limkimsan did you fix in the other branch?